Cesium3DTilesSelection/src/TilesetContentLoaderResult.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "TilesetContentLoader.h"
+
+#include <Cesium3DTilesSelection/ErrorList.h>
+#include <Cesium3DTilesSelection/Tile.h>
+#include <CesiumAsync/IAssetAccessor.h>
+#include <CesiumGeometry/Axis.h>
+
+#include <memory>
+#include <vector>
+
+namespace Cesium3DTilesSelection {
+struct LoaderCreditResult {
+  std::string creditText;
+
+  bool showOnScreen;
+};
+
+struct TilesetContentLoaderResult {
+  std::unique_ptr<TilesetContentLoader> pLoader;
+
+  std::unique_ptr<Tile> pRootTile;
+
+  CesiumGeometry::Axis gltfUpAxis{CesiumGeometry::Axis::Y};
+
+  std::vector<LoaderCreditResult> credits;
```

Comment:
Same here. Is this used?

Reply:
this will be used in the credit PR https://github.com/CesiumGS/cesium-native/pull/521. I leave this comment here to mark this as temporarily resolved until the credit PR is merged